src/components/KPrompt/KPrompt.vue

```diff
@@ -167,18 +168,18 @@ const close = (): void => {
   emit('canceled')
 }
 
-const proceed = (evt: any): void => {
+const proceed = (evt: Event): void => {
   if (disableProceedButton.value) return
 
   confirmationInput.value = ''
   emit('proceed', evt)
 }
 
-const handleKeydown = (e: any) => {
+const handleKeydown = (e: KeyboardEvent) => {
   if (props.isVisible) {
-    if (e.keyCode === 27) { // 'esc' key
+    if (e.key === 'Escape') {
       close()
-    } else if (e.keyCode === 13) { // 'enter' key
+    } else if (e.key === 'Enter') {
```

Comment:
question: is there a reason you're changing from the keycode? I'm not sure this is ideal even with/without the event typing